android-components/plugins/dependencies/src/main/java/ExtraRepositories.kt

```diff
@@ -0,0 +1,20 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// Extra Maven repositories to use
+object ExtraRepositories {
+    // The lines below are what we should be doing...
+    // val mozillaStaging = ApplicationServicesConfig.channel == ApplicationServicesChannel.STAGING
+    // val mozillaNightly = ApplicationServicesConfig.channel == ApplicationServicesChannel.NIGHTLY
+    // val mozillaNightlyStaging = ApplicationServicesConfig.channel == ApplicationServicesChannel.NIGHTLY_STAGING
+    //
+    // However, app-services doesn't yet have access to the maven nighlty buckets, so we use staging instead
```

Comment:
What's interesting here is that GeckoView (Nightly) isn't published to `nightly.mozilla.maven.org`. Distinct artifacts are published for nightly|beta|release: 
https://github.com/mozilla-mobile/firefox-android/blob/main/android-components/plugins/dependencies/src/main/java/Gecko.kt#L26

See: https://maven.mozilla.org/?prefix=maven2/org/mozilla/geckoview/

https://nightly.maven.mozilla.org/ has actually only ever been used by A-C. Do you have a preference here? It seems the former would make this a bit easier, as we could use the same approach as Gecko.kt. Not sure though, wdyt?

Reply:
OK, I don't see an advantage in using a separate Maven repo for Nightly. So following the same approach we use for GeckoView (Gecko.kt) makes sense to me, and should simplify this for you.